tests/ui/specs/MediaPlayer/MediaPlayer-specs.js

```diff
@@ -0,0 +1,76 @@
+const Page = require('./MediaPlayerPage');
+
+describe('MediaPlayer', function () {
+
+	beforeEach(function () {
+		Page.open();
+	});
+
+	describe('default', function () {
+		const mediaPlayerDefault = Page.components.mediaPlayerDefault;
+
+		it('should have the slider knob focused', function () {
+			expect(mediaPlayerDefault.slider.isFocused()).to.be.true();
+		});
+
+		const mediaPlayerDisabled = Page.components.mediaPlayerDisabled;
+
+		it('should have the slider knob focused', function () {
```

Comment:
can you move this test to another section ar update the title of the test to indicate it involves a disabled mediaplayer?